apiml/src/main/java/org/zowe/apiml/ZaasSchemeTransformApi.java

```
@@ -200,6 +224,10 @@ private static class RequestCredentialsHttpServletRequestAdapter implements Http
         @Delegate(excludes = Exclude.class)
         private HttpServletRequest request;
```

It looks it's not initialized, and could cause NPE when calling the upgrade method. `upgrade` method could rather throw `UnsupportedOperationException`.